Week3/homework/3-step3.js

```diff
@@ -3,6 +3,15 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
+  let x = "";
+  if (num <= 0) {
+    str = "";
+  } else {
+    for (let i = 0; i < num; i++) {
+      x += str;
+    }
+    str = x;
```

Comment:
In my re-reviewing your homework you don't need this `x` you can do all things with the `str` it self

Reply:
Oh ... yes you have right 😅😅